app/assets/javascripts/angular/services/repo_service.js

```diff
@@ -0,0 +1,17 @@
+app.service('Repo', ['$resource', function($resource) {
+    this.getRepoData = function(owner, repo){
+    var repo = $resource('/api/v1/repos', {owner: "@owner", repo: "@repo"});
+    .success(function (data){
+
+    })
+    .error(function (data, status) {
+      if (status === 404) {
+        $scope.error = 'That repository does not exist';
+      } else {
+        $scope.error = 'Error: ' + status;
+      }
+    });
+
+    return repo.query(owner: owner, repo: repo )
```

Comment:
Expected ')' and instead saw ':'.<br>Missing semicolon.<br>Expected an assignment or function call and instead saw an expression.<br>Expected '}' to match '{' from line 2 and instead saw ':'.<br>Expected an identifier and instead saw ')'.<br>'repo' is not defined.